TheJumpCloud/support
# NB-11628: Mac Users Not Having Ownership of The PWM Desktop App

## Issues
* [NB-11628](https://jumpcloud.atlassian.net/browse/NB-11628) - Auto-updater Asking for Admins permission on Mac

## What does this solve?
- Auto-updater Asking for Admins permission on Mac
- Mac Users Not Having Ownership of The PWM Desktop App

## Is there anything particularly tricky?
No

## How should this be tested?
1- Have multiple users (admin or standard) on your testing macOS machine.
2- Take the command from `Powershell/JumpCloud Commands Gallery/Mac Commands/Mac - Install JumpCloud Password Manager App.md`.
3- Create a file on your machine and name it whatever you want but the extension should be `.sh`. e.g: `example.sh`.
4- Paste the command you copied in step 1 in the `.sh` file you created in step 2.
5- Open your terminal and go to the directory where the `.sh` file you created in step 2 can be found.
6- Run `chmod 777 <scriptFileName>.sh` or `sudo chmod 777 <scriptFileName>.sh` depending on your situation.
7- Run `./<scriptFileName>.sh` **with admin rights**.
8- After step 6 finishes execution, an app will show on your desktop. Double click it and see if the app opens successfully.
9- Login to another User and go to context menu of `/Users/<username>/Applications/JumpCloud Password Manager.app` and click `Get Info`
10- The active user should have `Read & Write` permissions

## Screenshots
N/A

[NB-11628]: https://jumpcloud.atlassian.net/browse/NB-11628?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
